src/components/Cards/EventCard/EventCard.css

```diff
@@ -56,7 +61,7 @@
   background: radial-gradient(
     621px at 25.3% 13.8%,
     rgb(255, 255, 255) 0%,
-    rgb(234, 236, 255) 90%
+    rgb(255, 255, 255) 90%
```

Comment:
In the comment, I told you to use the colors and values from `theme.tsx`. This will help us to keep the theme consistent.